nssp/delphi_nssp/constants.py

```diff
@@ -0,0 +1,29 @@
+"""Registry for variations."""
+
+GEOS = [
+    "nation",
+    "state",
+    "county",
+]
+
+METRICS = ['percent_visits_covid','percent_visits_influenza',
+           'percent_visits_rsv','percent_visits_combined',
+           'percent_visits_smoothed_covid','percent_visits_smoothed_influenza',
+           'percent_visits_smoothed_rsv','percent_visits_smoothed_combined']
```

Comment:
question: what's the difference between these smoothed signals here ("percent_visits_smoothed_covid") vs below ("smoothed_percent_visits_covid")? Should the names match?

Reply:
suggestion: if we really do intend to rename "percent_visits_smoothed_covid" -> "smoothed_percent_visits_covid" at some point, please use a dictionary to keep the mapping straight. Relying on these two lists to have the same order will make it easy to introduce bugs, especially if these signal lists get longer in the future.

Reply:
fixed. https://github.com/cmu-delphi/covidcast-indicators/pull/1952/commits/95bdac15983b7abaefee95e5093a374ecac4ee94